src/applications/simple-forms/20-10207/pages/nonVeteranIdInfo.js

```diff
@@ -1,20 +1,23 @@
 import {
-  ssnOrVaFileNumberSchema,
-  ssnOrVaFileNumberUI,
+  ssnOrVaFileNumberNoHintUI,
+  ssnOrVaFileNumberNoHintSchema,
```

Comment:
I thought we were  no longer accepting VA file numbers at all. Is there no component for just ssn?